### ISSUE TITLE
Proposed: translation to French

### ISSUE THREAD

**Issue by notabene** [genmon/aboutfeeds#33]
Hi @genmon 

I'd be willing to translate `index.md` to French, if you're OK with it.
Would `index.fr.md` be a good file name at the project root, or do you have plans to add `/fr/` etc. to the URL tree for each translation?

**Comment by ricveal** [comment]
Good idea! If @genmon agrees with adding translations, I would be happy to translate it into Spanish (es) too.
